TWLight/users/management/commands/user_check.py

```diff
@@ -1,16 +1,13 @@
-import json
 import logging
-import urllib.request, urllib.error, urllib.parse
 
-from datetime import datetime
-from django.conf import settings
-from django.db import models
 from django.contrib.auth.models import User
-from django.core.management.base import BaseCommand, CommandError
-from ....users.models import Editor, UserProfile
+from django.core.management.base import BaseCommand
+from TWLight.users.helpers.editor_data import editor_global_userinfo
 
 logger = logging.getLogger(__name__)
 
+# TODO: Check if the assertion inside editor_global_userinfo bubbles up to this command.
```

Comment:
Is this still outstanding for this PR, or later?

Reply:
later